entity-service/src/test/kotlin/com/egm/stellio/entity/authorization/Neo4jAuthorizationRepositoryTest.kt

```diff
@@ -613,6 +616,139 @@ class Neo4jAuthorizationRepositoryTest : WithNeo4jContainer {
         neo4jRepository.deleteEntity(apiaryUri)
     }
 
+    @Test
+    fun `it should return matching entities that user have rights`() {
+        val userEntity = createEntity(userUri, listOf(USER_TYPE), mutableListOf())
+        val firstEntity = createEntity(
+            "urn:ngsi-ld:Beekeeper:1230".toUri(),
+            listOf("Beekeeper"),
+            mutableListOf(Property(name = expandedNameProperty, value = "Scalpa"))
```

Comment:
do you really need the `name` properties in your entities? it is not used in queries, nor in results.

Reply:
I removed it